scripts/variantstore/beta_docs/gvs-exome-workspace-description.md

```diff
@@ -42,6 +42,7 @@ Note that, by default, the Exomes Beta workflow uses the Blended Genomes Interva
 The following files are stored in the Google Cloud Storage path specified in the `extract_output_gcs_dir` workflow input.
 
 - Sharded joint VCF files, index files, the interval lists for each sharded VCF, and a list of the sample names included in the callset.
```

Comment:
See gvs_outputs.md for the expected number of shards?

Reply:
I will add a link, good call!